css/index.css

```diff
@@ -137,10 +139,40 @@ footer nav {
     justify-content: space-between;
     align-items: center;
     padding: 20px 2%;
-    font-size: 14px;
+    font-size: 0.875rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+@media (max-width: 1000px) {
```

Comment:
I notice you added another class to specify even boxes(i.e.,  .box2 - .box10) . Did you add these classes in your html too?